src/Tizen.NUI/src/public/BaseComponents/GLView.cs

```diff
@@ -57,7 +57,18 @@ internal GLView(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemor
         /// </summary>
         /// <param name="colorFormat">The format of the color buffer</param>
         /// <since_tizen> 10 </since_tizen>
-        public GLView(ColorFormat colorFormat) : this(Interop.GLView.New((int)colorFormat), true)
+        public GLView(ColorFormat colorFormat) : this(Interop.GLView.New((int)colorFormat, (int)GLBackendMode.Default), true)
```

Comment:
I think that the reason the review bot tagged "ACR required" is that you changed this API.